ze/tracer_ze_helpers.include.c

```diff
@@ -750,10 +752,258 @@ static inline void _dump_memory_info(ze_command_list_handle_t hCommandList, cons
   }
 }
 
+////////////////////////////////////////////
+
+#define _ZE_ERROR_MSG(NAME,RES) {fprintf(stderr,"%s() failed at %d(%s): res=%x\n",(NAME),__LINE__,__FILE__,(RES));}
+#define _ZE_ERROR_MSG_NOTERMINATE(NAME,RES) {fprintf(stderr,"%s() error at %d(%s): res=%x\n",(NAME),__LINE__,__FILE__,(RES));}
+#define _ERROR_MSG(MSG) {perror((MSG)); fprintf(stderr,"errno=%d at %d(%s)",errno,__LINE__,__FILE__);}
+
+static int initialized = 0;
+
+#define MAX_N_DEVS  (8)
```

Comment:
Maybe add a `TODO` to said we should refactor to avoid magic numbers / static allocation

Reply:
Re-implemented the block in 3930be7. No magic numbers.